src/DetailsView/actions/details-view-action-message-creator.ts

```diff
@@ -384,6 +384,11 @@ export class DetailsViewActionMessageCreator extends DevToolActionMessageCreator
         this.dispatcher.sendTelemetry(TelemetryEvents.DETAILS_VIEW_OPEN, telemetryData);
     }
 
+    public onIframePermissionWarningShown = () => {
```

Comment:
Would it be better to name this like the `ScanIncompleteWarning` pattern and include which warnings were present in the telemetryData?

Reply:
I think I'm going to remove the onRender functionality and that likely means this is not necessary.